rts/motoko-rts/src/memory/ic/linear_memory.rs

```diff
@@ -6,18 +6,18 @@ use crate::types::*;
 /// Amount of garbage collected so far.
 pub(crate) static mut RECLAIMED: Bytes<u64> = Bytes(0);
 
-// Heap pointer
+// Heap pointer (skewed)
 extern "C" {
-    pub(crate) fn setHP(new_hp: u32);
+    pub(crate) fn setHP(new_hp: u32); // usize????
```

Comment:
Maybe it is better to use `Value` if it is a skewed pointer?

Reply:
This is a local function now.

Reply:
and `setHP` takes an `usize`. 27409cc3b74b517c6051a362a59a99ceb455d0a0